ruleset/rules/0905-cisco-ftd_rules.xml

```diff
@@ -1,230 +1,234 @@
-        <if_matched_sid>91501</if_matched_sid>
-        <description>Multiple FTD alert messages.</description>
-        <group>pci_dss_10.6.1,pci_dss_11.4,gpg13_4.12,gdpr_IV_35.7.d,tsc_CC7.2,tsc_CC7.3,tsc_CC6.1,tsc_CC6.8,</group>
-    </rule>
-
-    <rule id="91527" level="10" frequency="8" timeframe="360">
-        <if_matched_sid>91502</if_matched_sid>
-        <description>FTD: Multiple critical messages.</description>
-        <group>pci_dss_10.6.1,pci_dss_11.4,gpg13_4.1,gdpr_IV_35.7.d,tsc_CC7.2,tsc_CC7.3,tsc_CC6.1,tsc_CC6.8,</group>
-    </rule>
-
-    <rule id="91528" level="10" frequency="10" timeframe="120">
-        <if_matched_sid>91503</if_matched_sid>
-        <description>FTD: Multiple error messages.</description>
-        <group>system_error,pci_dss_10.6.1,pci_dss_11.4,gpg13_4.3,gdpr_IV_35.7.d,tsc_CC7.2,tsc_CC7.3,tsc_CC6.1,tsc_CC6.8,</group>
-    </rule>
-
-    <rule id="91529" level="10" frequency="10" timeframe="120">
-        <if_matched_sid>91504</if_matched_sid>
-        <description>FTD: Multiple warning messages.</description>
-        <group>pci_dss_10.6.1,gpg13_4.12,gdpr_IV_35.7.d,tsc_CC7.2,tsc_CC7.3,</group>
-    </rule>
-
-    <rule id="91530" level="8">
-        <if_sid>91502</if_sid>
-        <field name="event.id">106016</field>
-        <description>FTD: IP spoofing attack detected.</description>
-        <group>gdpr_IV_35.7.d,</group>
-    </rule>
-</group>
\ No newline at end of file
+
+  <!-- General rule -->
+  <rule id="91500" level="0">
+    <decoded_as>cisco-ftd</decoded_as>
+    <description>cisco-ftd rules</description>
+  </rule>
+
+  <rule id="91501" level="7">
+    <if_sid>91500</if_sid>
+    <field name="event.severity">1</field>
+    <description>FTD: alert message.</description>
+  </rule>
+
+  <rule id="91502" level="5">
+    <if_sid>91500</if_sid>
+    <field name="event.severity">2</field>
+    <description>FTD: critical message.</description>
+  </rule>
+
+  <rule id="91503" level="4">
+    <if_sid>91500</if_sid>
+    <field name="event.severity">3</field>
+    <description>FTD: error message.</description>
+  </rule>
+
+  <rule id="91504" level="3">
+    <if_sid>91500</if_sid>
+    <field name="event.severity">4</field>
+    <description>FTD: warning message.</description>
+  </rule>
+
+  <rule id="91505" level="2">
+    <if_sid>91500</if_sid>
+    <field name="event.severity">5|6</field>
+    <description>FTD: notification/informational message.</description>
+  </rule>
+
+  <rule id="91506" level="0">
+    <if_sid>91500</if_sid>
+    <field name="event.severity">7</field>
+    <description>FTD: debug message.</description>
+  </rule>
+
+  <rule id="91507" level="9">
+    <if_sid>91505</if_sid>
+    <field name="event.id">605004</field>
+    <description>FTD: Failed login attempt.</description>
+    <group>authentication_failed,gpg13_7.8,gdpr_IV_35.7.d,gdpr_IV_32.2,pci_dss_10.2.4,pci_dss_10.2.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
```

Comment:
Reorder these groups to locate GDPR controls before GPG. Same for the rest of the rules.